test/Interop/lit.local.cfg

```diff
@@ -44,9 +44,22 @@ else:
     config.substitutions.insert(0, ('%target-abi', 'SYSV'))
     config.substitutions.insert(0, ('%target-swift-flags', ''))
 
+# target_sdk_libcxx_path is empty when targeting non-Darwin platforms
+config.target_libcxx_path = config.target_sdk_libcxx_path
+
 if get_target_os() in ['linux-gnu']:
-    if os.path.exists('/usr/include/c++/v1') or os.path.exists('/usr/local/include/c++/v1'):
-        config.available_features.add('system_wide_libcxx')
+    for p in ['/usr/include/c++/v1', '/usr/local/include/c++/v1']:
```

Comment:
This is the path of libc++ on Linux, but we're compiling with libstdc++ on Linux by default, so I think we're checking the wrong path here.

The path to `<span>` in libstdc++ would look like `/usr/include/c++/13/span`.